include/exadg/convection_diffusion/user_interface/parameters.cpp

```diff
@@ -508,6 +532,8 @@ Parameters::print_parameters_spatial_discretization(dealii::ConditionalOStream c
 
   print_parameter(pcout, "Polynomial degree", degree);
 
+  print_parameter(pcout, "Enable adaptivity", enable_adaptivity);
+
```

Comment:
I think our standard procedure woul be to print the AMR parameters `if(enable_adaptivity)`.